### ISSUE TITLE
Can't install on Ubuntu 18.04

### ISSUE THREAD

**Issue by thomasdeniau** [kelvin13/atomic-blonde#1]
Hi, 

When trying to install this package with apm on Ubuntu 18,04 I get:

```
In file included from /home/swercadmin/.atom/.node-gyp/.cache/node-gyp/6.1.12/include/node/node.h:63:0,
                 from ../node_modules/nan/nan.h:56,
                 from ../cpp/blonde.cpp:4:
/home/swercadmin/.atom/.node-gyp/.cache/node-gyp/6.1.12/include/node/v8.h:6537:23: note: candidate: static v8::Local<v8::Value> v8::Exception::TypeError(v8::Local<v8::String>)
   static Local<Value> TypeError(Local<String> message);
                       ^~~~~~~~~
/home/swercadmin/.atom/.node-gyp/.cache/node-gyp/6.1.12/include/node/v8.h:6537:23: note:   no known conversion for argument 1 from ‘v8::MaybeLocal<v8::String>’ to ‘v8::Local<v8::String>’
../cpp/blonde.cpp:310:69: error: no matching function for call to ‘v8::Exception::Error(v8::MaybeLocal<v8::String>)’
                 v8::String::NewFromUtf8(isolate, "invalid response")));
```

[errors.log](https://github.com/kelvin13/atomic-blonde/files/5598468/errors.log)

**Comment by tayloraswift** [comment]
this should be resolved in the latest 1.0.5 release

**Comment by mcstrassell09** [comment]
is there support for macOS/UNIX systems in the works?
